site/src/testHelpers/handlers.ts

```diff
@@ -438,4 +444,8 @@ export const handlers = [
   rest.get("/api/v2/workspaceagents/:agent/listening-ports", (_, res, ctx) => {
     return res(ctx.status(200), ctx.json(M.MockListeningPortsResponse));
   }),
+
+  rest.get("/api/v2/integrations/jfrog/xray-scan", (_, res, ctx) => {
+    return res(ctx.status(404));
+  }),
```

Comment:
Why did we add these?

Reply:
To avoid getting unhandled request warnings during tests. The warning was introduced many PRs ago but I only noticed it now since it was messing with the output for my tests so I just fixed it.